packages/campaignclassic/README.md

```diff
@@ -0,0 +1,170 @@
+# React Native AEP Campaign Classic Extension
+
+[![npm version](https://badge.fury.io/js/%40adobe%2Freact-native-aepcampaignclassic.svg)](https://www.npmjs.com/package/@adobe/react-native-aepcampaignclassic)
+[![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aepcampaignclassic)](https://www.npmjs.com/package/@adobe/react-native-aepcampaignclassic)
+
+`@adobe/react-native-aepcampaignclassic` is a wrapper around the iOS and Android [Adobe Experience Platform Campaign Classic Extension](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-campaignclassic) to allow for integration with React Native applications.
+
+## Peer Dependencies
+
+The Adobe Experience Platform Campaign Classic extension has the following peer dependency, which must be installed prior to installing the optimize extension:
+
+- [Core](../core/README.md)
+
+## Installation
+
+See [Requirements and Installation](https://github.com/adobe/aepsdk-react-native#requirements) instructions on the main page
+
+Install the `@adobe/react-native-aepcampaignclassic` package:
+
+NPM:
+
+```bash
+npm install @adobe/react-native-aepcampaignclassic
+```
+
+Yarn:
+
+```bash
+yarn add @adobe/react-native-aepcampaignclassic
+```
+
+## Usage
+
+### Initializing and registering the extension
+
+Initialization of the SDK should be done in native code, documentation on how to initialize the SDK can be found [here](https://github.com/adobe/aepsdk-react-native#initializing).
+
+Example:
+
+iOS
+
+```objectivec
+@import AEPCore;
+@import AEPLifecycle;
+@import AEPCampaignClassic;
+
+@implementation AppDelegate
+-(BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  [AEPMobileCore setLogLevel: AEPLogLevelTrace];
+  [AEPMobileCore registerExtensions: @[ AEPCampaignClassic.class] completion:^{
+    [AEPMobileCore configureWithAppId:@"yourAppID"];
+    [AEPMobileCore lifecycleStart:@{@"contextDataKey": @"contextDataVal"}];
+  }
+  ];
+  return YES;
+}
+@end
+```
+
+Android
+
+```java
+import com.adobe.marketing.mobile.AdobeCallback;
+import com.adobe.marketing.mobile.InvalidInitException;
+import com.adobe.marketing.mobile.Lifecycle;
+import com.adobe.marketing.mobile.LoggingMode;
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.campaignclassic.CampaignClassic;
```

Comment:
Since we are going to use Android 1.0, you can revert the import to 
`import com.adobe.marketing.mobile.CampaignClassic;`